### DISCUSSION TITLE
Registration of a command from a bundle

### DISCUSSION THREAD

**Discussion by chregi** [pimcore/pimcore#13903] [Q&A / Support]
I have created a bundle. I have created in the subdirectory "Command" of a new class SyncCommand (extends AbstractCommand). How do I register this server to see the via the console "bin/console list"?

**Comment by jdreesen** [comment]
https://symfony.com/doc/5.4/console.html#registering-the-command

**Comment by mattamon** [comment]
Hey @chregi,

See the official documentation [here](https://pimcore.com/docs/pimcore/current/Development_Documentation/Development_Tools_and_Details/Console_CLI.html)

The most important part is, that your bundle is registered via the ``` config/bundles.php```
Implementing the configure function should register your command.


```php
 protected function configure()
    {
        $this
            ->setName('awesome:command')
            ->setDescription('Awesome command');
    }
```

Check out the [customer-data-framework BuildSegmentsCommand](https://github.com/pimcore/customer-data-framework/blob/4.x/src/Command/BuildSegmentsCommand.php) as a reference.
